storage/remote/store.go

```diff
@@ -0,0 +1,144 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+	"github.com/onflow/flow-archive/api/archive"
+	"github.com/onflow/flow-archive/codec/zbor"
+	"github.com/onflow/flow-emulator/storage"
+	"github.com/onflow/flow-emulator/storage/sqlite"
+	exeState "github.com/onflow/flow-go/engine/execution/state"
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/fvm/storage/snapshot"
+	"github.com/onflow/flow-go/ledger/common/pathfinder"
+	"github.com/onflow/flow-go/ledger/complete"
+	flowgo "github.com/onflow/flow-go/model/flow"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type Store struct {
+	*sqlite.Store
+	client archive.APIClient
+}
+
+func New(chainID flowgo.ChainID) (*Store, error) {
+	archiveHosts := map[flowgo.ChainID]string{
+		flowgo.Mainnet: "archive.mainnet.nodes.onflow.org:9000",
+		flowgo.Testnet: "archive.testnet.nodes.onflow.org:9000",
```

Comment:
It might make sense to make the archive host a configuration option and use these values as defaults

Reply:
Yep

Reply:
maybe we can move this to constructor also